proto/cosmos/distribution/v1beta1/authz.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package cosmos.staking.v1beta1;
```

Comment:
adjust package name -- also if we adjust this, should it be something like `v1beta2`?

```suggestion
package cosmos.distribution.v1beta1;
```